src/AutoMapper/QueryableExtensions/BuilderException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AutoMapper.QueryableExtensions
+{
+    public class BuilderException : Exception
```

Comment:
I don't think we need a new exception type here. AutoMapperMappingException should do. Just pass it the types so it's easy to understand what's going on.

Reply:
Uses now `AutoMapperMappingException`